code/modules/client/verbs/who.dm

```diff
@@ -92,5 +84,11 @@
 				continue //Don't show afk admins to adminwho
 			if(!C.holder.fakekey)
 				msg += "\t[C] is a [C.holder.rank]\n"
+	if(length(GLOB.mentors) > 0)
+		msg += "<b>Mentors:</b> \n"
+		for(var/client/C in sortList(GLOB.clients))
+			var/mentor = GLOB.mentor_datums[C.ckey]
+			if(mentor)
+				msg += "<span>\t[C.key]</span> is a Mentor \n"
```

Comment:
You want to remove that `<span>` and `</span>`. They serve no purpose if they're empty.